server/functions/tsconfig.json

```
@@ -0,0 +1,15 @@
+{
+  "compilerOptions": {
+    "module": "commonjs",
```

Is there a reason to use commonjs over es2020?
```suggestion
    "module": "es2020",
```

---

Google's default Node.js version at the moment is 12, which doesn't do `es2020` modules yet. [Here's a helpful discussion](https://stackoverflow.com/questions/59787574/typescript-tsconfig-settings-for-node-js-12).